### PR TITLE
Separate out integrations docs

### PR DESCRIPTION
**Pull request by dtuite**
https://app.shortcut.com/larder/story/7999/separate-out-integrations

### CODE REVIEW

**Comment by iain-b** [comment]
Can David read my thoughts ? 🤔 I was just looking at our docs for google auth (regarding the gcalendar plugin) and was thinking we could do this ^^

**Review by Xantier — APPROVED** [review]

File: content/docs/docs-nav.yaml [hunk 1]
```diff
@@ -43,6 +39,11 @@ nav:
     - Travis CI: '/docs/integrations/travis-ci/'
     - Tech radar: '/docs/configuration/tech-radar/'
     - Cortex: '/docs/integrations/cortex'
+  - Integrations:
+    - Google Cloud Platform: '/docs/integrations/gcp/'
```
Comment (Xantier) [inline content/docs/docs-nav.yaml:43]: Move this to be second to last to order alphabetically.

**Comment by dtuite** [comment]
@iain-b S3 docs can go into this section now.

File: content/docs/docs-nav.yaml (hunk 1 above)
Comment (dtuite) [reply content/docs/docs-nav.yaml]: Done

**Review by Xantier — APPROVED** [review]

**Pull request merged**